command_walkbox.go

```diff
@@ -0,0 +1,12 @@
+package pctk
+
+// EnableWalkBox is a command that will enable or disable a walkbox in the room.
+type EnableWalkBox struct {
+	WalkBox *WalkBox
+	Enabled bool
+}
+
+func (cmd EnableWalkBox) Execute(app *App, done *Promise) {
+	cmd.WalkBox.room.wbmatrix.EnableWalkBox(cmd.WalkBox.walkBoxID, cmd.Enabled)
+	done.Complete()
+}
```

Comment:
It's better to use the new style of command definition using the `CommandFunc` type and declaring functions that returns it.

Reply:
Done! :D